.github/workflows/nightly.yml

```diff
@@ -117,10 +117,95 @@ jobs:
         run: |
           cd "${BUILD_DIR}"
           cmake --build "${BUILD_DIR}" --target package
+      - name: Get version info
+        run: |      
+          cd ${SRC_DIR}
+          echo "VERSION=$(git describe --tags | sed 's/^release-//;s/-/+/;s/-/~/;s/rc/~rc/')-${MSYSTEM_CARCH}" >> $GITHUB_ENV
```

Comment:
```suggestion
          echo "VERSION=$(git describe --tags | sed 's/^release-//;s/-/+/;s/-/~/;s/rc/~rc/')" >> $GITHUB_ENV
          [[ ${MSYSTEM_CARCH} == x86_64 ]] && echo "SYSTEM=win64" || echo "SYSTEM=woa64" >> $GITHUB_ENV
```

Reply:
This matches the "SYSTEM" suffix to what CPack generates anyway (TDB for "woa64" though).

Reply:
👍

Reply:
Not sure why you set a new env variable SYSTEM here. Where is it used afterwards?

Reply:
For the upload step just below.

Reply:
ah, got it!